src/screening.cpp

```diff
@@ -693,4 +1039,75 @@ int main(int nargs, char **argv)
 		fclose(csv_data_file);
 		printf(convert_string("Screening finished for "+prefix+arg1+". Runtime: %.2f sec\n"), 1.0e-6*(walltime_usec() - start_usec) );
 	}
+
+	// Record the current walltime
+    t_usec = walltime_usec();
+
+    // Import reservoir data for grid square FIX THESE FILENAMES TO BE THOSE THAT ARE ACTUALLY OUTPUT BY OTHER CODE
+	if (display)
+		printf("Reading reservoir data for turkey nest optimisation in grid square\n");
+
+	vector<RoughReservoir> existing_reservoirs_gs;
+	vector<RoughReservoir> pit_reservoirs_gs;
+	vector<RoughReservoir> ocean_reservoirs_gs;
+	vector<RoughReservoir> reservoirs_gs;
```

Comment:
Would it be possible to do the turkeys nest search without relying on the inputs from a bunch of other screening runs?

Reply:
Turkey nest screening is performed in a search area around each of the previously located reservoirs to minimise screening time - turkey nest sites are not expected to be paired with each other, so no value in finding rough turkey nests far away from other eligible sites. Probably worth actually defining a TN screening according to an argument to the main function though, rather than having it take place at the end of other screening runs as it does currently